### PR TITLE
Adding Timezone, Page, and Screen to Outbound Integration

### PR DESCRIPTION
**Pull request by Tang8330**

### CODE REVIEW

**Comment by Tang8330** [comment]
@stevenmiller888 @f2prateek @amillet89

File: lib/mapper.js [hunk 1]
```diff
@@ -93,3 +98,43 @@ exports.alias = function(alias){
...
+};
+
+/**
+ * Map screen.
+ *
+ * @param {Screen} screen
+ * @return {Object}
+ * @api private
+ */
+exports.screen = function(screen){
```
Comment (f2prateek) [inline lib/mapper.js:124]: This logic looks mostly the same as the `page` call, except for the name. It should be pulled into a function.

File: lib/mapper.js [hunk 2]
```diff
@@ -22,6 +22,11 @@ exports.identify = function(identify){
 
   var attrs = body.attributes;
 
+  if (identify.timezone()) {
```
Comment (f2prateek) [inline lib/mapper.js:25]: In the future please send seperate logical changes as seperate PRs.

File: lib/index.js [hunk 3]
```diff
@@ -1,15 +1,12 @@
-
```
Comment (f2prateek) [inline lib/index.js:1]: Please don't change our code styles/formatting/whitespaces, especially when there are code changes in the PR. You can send these as a seperate PR if you feel strongly about the changes.

**Comment by Tang8330** [comment]
@f2prateek addressed!

**Comment by f2prateek** [comment]
@Tang8330 Is this considered a breaking change? Will customers be upset if they suddenly start seeing page/screen calls show up in Outbound?

**Comment by Tang8330** [comment]
@f2prateek no. They have been asking for it.

File: lib/mapper.js [hunk 4]
```diff
@@ -93,3 +98,42 @@ exports.alias = function(alias){
...
+  var evtName = "[Segment Screen]";
+  if (screen.properties().name) {
+    evtName += " " + screen.properties().name;
+  }
+  return pageAndScreen(evtName, screen);
+};
+
+function pageAndScreen(evtName, pageOrScreen) {
+    return {
+        user_id: pageOrScreen.userId(),
```
Comment (Tang8330) [inline lib/mapper.js:134]: @f2prateek Hi Prateek. We have been noticing a weird behavior -- inside Screen View, there is no userId.
Do you know what may be going on with this integration code?

Comment (f2prateek) [reply lib/mapper.js:134]: User id is optional. Anonymous users may not have a userid.

**Pull request merged**